person.rb

```diff
@@ -0,0 +1,25 @@
+# Person Class
+class Person
+  def initialize(age,
+                 name = 'Unknown',
+                 _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  # rubocop:disable Naming/PredicateName
+  def is_of_age?
+    age >= 18
+  end
+
+  private :is_of_age?
+
+  # rubocop:enable Naming/PredicateName
+  def can_use_services?
+    is_of_age? || parent_permission ? true : false
```

Comment:
- [ ] When I define a **Person** instance with age less than 18, I get an error because `parent_permission` is not defined (see image). I would recommend you use the @parent_permission instance variable here instead, like so: `is_of_age? || @parent_permission`

  ![temitope-perm](https://user-images.githubusercontent.com/36542986/157084608-6ec9fe21-5c42-4622-afef-8162e3d1c671.jpg)